tests/collections/common/tokenizers/text_to_speech/test_tokenizer_utils.py

```diff
@@ -120,3 +121,30 @@ def test_any_locale_word_tokenize_with_numbers(self):
 
         output = any_locale_word_tokenize(input_text)
         assert output == expected_output
+
+    @pytest.mark.run_only_on('CPU')
+    @pytest.mark.unit
+    def test_french_text_preprocessing_lower(self):
+        input_text = "pomme banane poire"
+        expected_output = "pomme banane poire"
+
+        output = french_text_preprocessing(input_text)
+        assert output == expected_output
+
+    @pytest.mark.run_only_on('CPU')
+    @pytest.mark.unit
+    def test_french_text_preprocessing_mixed(self):
+        input_text = "BONJOUR le Monde!"
+        expected_output = "bonjour le monde!"
+
+        output = french_text_preprocessing(input_text)
+        assert output == expected_output
+
+    @pytest.mark.run_only_on('CPU')
+    @pytest.mark.unit
+    def test_french_text_preprocessing_upper(self):
+        input_text = "A BIENTÔT."
+        expected_output = "a bientôt."
+
+        output = french_text_preprocessing(input_text)
+        assert output == expected_output
```

Comment:
these tests are for `text_preprocessing` funcs, not for any tokenize funcs. Could you pls revise following the above unit tests?

Reply:
we can remove the above three unit tests, and instead add test examples inside all functions related to `any_locale_word_tokenize`. For example, you could extend below `input_text` and `expected_output` as a list by adding fr-fr examples.
```python
@pytest.mark.run_only_on('CPU')
    @pytest.mark.unit
    def test_any_locale_word_tokenize(self):
        input_text = "apple banana pear"
        expected_output = self._create_expected_output(["apple", " ", "banana", " ", "pear"])

        output = any_locale_word_tokenize(input_text)
        assert output == expected_output

    @pytest.mark.run_only_on('CPU')
    @pytest.mark.unit
    def test_any_locale_word_tokenize_with_accents(self):
        input_text = "The naïve piñata at the café..."
        expected_output = self._create_expected_output(
            ["The", " ", "naïve", " ", "piñata", " ", "at", " ", "the", " ", "café", "..."]
        )

        output = any_locale_word_tokenize(input_text)
        assert output == expected_output

    @pytest.mark.run_only_on('CPU')
    @pytest.mark.unit
    def test_any_locale_word_tokenize_with_numbers(self):
        input_text = r"Three times× four^teen ÷divided by [movies] on \slash."
        expected_output = self._create_expected_output(
            [
                "Three",
                " ",
                "times",
                "× ",
                "four",
                "^",
                "teen",
                " ÷",
                "divided",
                " ",
                "by",
                " [",
                "movies",
                "] ",
                "on",
                " \\",
                "slash",
                ".",
            ]
        )

        output = any_locale_word_tokenize(input_text)
        assert output == expected_output
```